plugins/woocommerce/templates/order/order-details.php

```diff
@@ -115,5 +146,5 @@
 do_action( 'woocommerce_after_order_details', $order );
 
 if ( $show_customer_details ) {
-	wc_get_template( 'order/order-details-customer.php', array( 'order' => $order ) );
+	wc_get_template( 'order / order - details - customer . php', array( 'order' => $order ) );
```

Comment:
whatever happened here is breaking the page :) 

<img width="793" alt="image" src="https://github.com/user-attachments/assets/96d503ae-7782-4011-b2be-614d64d6a9eb">

```suggestion
	wc_get_template( 'order/order-details-customer.php', array( 'order' => $order ) );
```